pkg/injection/namespace/bootstrapperconfig/bootstrapperconfig.go

```diff
@@ -88,11 +104,37 @@ func (s *SecretGenerator) GenerateForDynakube(ctx context.Context, dk *dynakube.
 		if err != nil {
 			return errors.WithStack(err)
 		}
+
+		// create cert secret for otlp configuration
+		// TODO is the creation of the OTLP related secrets tied to the same preconditions as the creation of the bootstrap secrets?
+		// TODO should we decouple the OTLP secret creation to be independent of any errors during the creation of the other secrets?
+		err = s.createSecretForNSlist(ctx, consts.OtlpIngestCertsSecretName, CertsConditionType, nsList, dk, certs)
+		if err != nil {
+			return errors.WithStack(err)
+		}
 	}
 
 	return nil
 }
 
+func extractApiToken(data map[string][]byte) map[string][]byte {
```

Comment:
see https://github.com/Dynatrace/dynatrace-operator/pull/5397/files#diff-62036e9d0edf8b3704940783898f28f99f2c1611e1da5461331811aab7db2462R74-R81 for why this is currently done this way

Reply:
can be solved an other way, see [comment](https://github.com/Dynatrace/dynatrace-operator/pull/5397/files#r2336269320)